compose/mpp/demo-uikit/README.md

```diff
@@ -1,9 +1,4 @@
-### Setup
- - in ~/.gradle/gradle.properties set: 
-```
-androidx.compose.multiplatformEnabled=true
-```
- - Open dir .../frameworks/support with AppCode, install plugin KMM for AppCode.
+ - Open dir ../frameworks/support with Android Studio, install KMM plugin.
```

Comment:
We still need AppCode for now to launch this demo.
Or use `xcodegen` tool in mpp/demo and launch in Xcode.

Reply:
Done